Spicing up the frontend

sloan-seamus opened the pull request:
Creating a new claim in New Claim will redirect to View Claims instead of Edit Page --Until loading issue is resolved--

Modified color scheme --No more "bruised" look
Rearranged header toolbar --Different menu to come soon

Commit:
Clicking on a claim in View Claims will take you to Edit Claims page
Creating a new claim in New Claim will redirect to View Claims instead of Edit Page --Until loading issue is resolved--

Modified color scheme --No more "bruised" look
Rearranged header toolbar --Different menu to come soon

sloan-seamus commented on frontend/src/app/header/header.component.html:
```diff
@@ -1,9 +1,11 @@
 <header class="head nav-bar">
     <ly-toolbar position="relative" bg="primary" [elevation]="5" class="auto-height">
-      <img src="../../assets/ford-logo.png" alt="Logo" class="center-to-page width-100 homepage-logo" routerLink="/home">
+      <img src="../../assets/ford-logo.png" alt="Logo" class="center-to-page width-100 homepage-logo vspacing" routerLink="/home">
```
I need a little help with this. I think I'm stupid.

I want a little bit of spacing between the top/bottom of the Ford logo and the toolbar, but padding and margins don't seem to do anything??

ej3076 replied:
@sloan-seamus you're adding a class called `vspacing` but that's not defined anywhere...

use the existing margin classes - if you want more space above / below on the toolbar add these: `mtm mbm`
margin-top-medium, margin-bottom-medium, these are defined in styles.scss

ej3076 commented on frontend/src/app/home/home.component.html:
```diff
@@ -2,13 +2,20 @@
-  </a>
+
+  <button ly-button raised outlined color="primary" size="large" lyType="title" class="width-100 mth mbh" routerLink="/claims/new"> Start a New Claim </button>
+<!--
```
Remove all these too

ej3076 commented on frontend/src/app/middleware/middleware.service.ts:
```diff
@@ -83,7 +83,7 @@ export class MiddlewareService {
         success => {
           // Log when successful / errored navigation
-          this.router.navigate([`/claims/${claim.vehicle.vin}`]).then(
+          this.router.navigate([`/claims`]).then(
```
You can probably undo this as well - in my PR i have a little dumb workaround for the data not showing.

sloan-seamus replied:
I left it for the time being so my branch is still functional

ej3076 replied:
Revert this - my branch fixes it temporarily

ej3076 commented on frontend/src/app/view-claims/view-claims.component.html:
```diff
@@ -4,14 +4,14 @@
   <ly-tab>
     <button ly-tab-label>Awaiting Settlement Option</button>
-    <ly-grid *ngIf="sortClaims(0).length > 0; else noClaims" container [spacing]="16">
+    <ly-grid *ngIf="sortClaims(0).length > 0; else hasClaims" container [spacing]="16">
```
This looks like you check if there is any claims, and if not... go to `hasClaims`? What is this doing?

ej3076 commented on frontend/src/app/view-claims/view-claims.component.html:
```diff
@@ -49,7 +49,7 @@ <h3 [lyTyp]="'title'">{{ claim.vehicle.vin }}</h3>
   <ly-tab>
     <button ly-tab-label (click)="sortClaims(3)">Claim Resolved</button>
-    <ly-grid *ngIf="sortClaims(3).length > 0; else noClaims" container [spacing]="16">
+    <ly-grid *ngIf="sortClaims(3).length > 0; then hasClaims else noClaims" container [spacing]="16">
```
Same here - what is this doing?

sloan-seamus replied:
That was something I was trying out and forgot to delete

ej3076 reviewed (changes requested):
Just a few comments.

dsifford reviewed (approved):
Nothing further to add.

Commit:
Home Page Cleanup

Added a carousel (just pitching the idea)
Larger buttons for Start/View claims
Larger Header -with larger logo, logging, and menu buttons

Commit:
Home Page Cleanup

Added a carousel (just pitching the idea)
Larger buttons for Start/View claims
Larger Header -with larger logo, logging, and menu buttons

sloan-seamus committed:
Merge branch 'master' into frontend_beautification

Commit:
Merge branch 'frontend_beautification' of https://github.com/ej3076/total-loss-process into frontend_beautification

Commit:
Merge branch 'frontend_beautification' of https://github.com/ej3076/total-loss-process into frontend_beautification

sloan-seamus committed:
Merge branch 'master' into frontend_beautification

ej3076 commented on frontend/src/app/header/header.component.html:
```diff
@@ -1,24 +1,30 @@
+
+<header class="head mbm">
+    <ly-toolbar position="relative" bg="primary" [elevation]="5" class="header-height">
+      <a href="/home">
```
Change this from `href` to `routerLink`

ej3076 reviewed (changes requested):
Couple of changes

Commit:
Merge branch 'frontend_beautification' of https://github.com/ej3076/total-loss-process into frontend_beautification

ej3076 reviewed (approved):
Looks good
